tests/unit/test_truenas_verify.py

```diff
@@ -0,0 +1,79 @@
+import os
+import pytest
+import subprocess
+import time
+
+from truenas_api_client import Client
+
+
+VERIFY_LOG_PREAMBLE = '/var/log/audit/truenas_verify'
+DEFAULT_VERIFY_LOG_NAME = f"{VERIFY_LOG_PREAMBLE}.log"
+CUSTOM_NAME = "init_test"
+CUSTOM_VERIFY_LOG_NAME = f"{VERIFY_LOG_PREAMBLE}.{CUSTOM_NAME}.log"
+
+
+def delete_file(fname):
+    try:
+        os.unlink(fname)
+    except FileNotFoundError:
+        pass
+
+
+@pytest.fixture(scope='module')
+def verify_setup():
+    # Setup:  Remove existing default log file
+    delete_file(DEFAULT_VERIFY_LOG_NAME)
+
+    try:
+        yield os.path.exists(DEFAULT_VERIFY_LOG_NAME)
+    finally:
+        # Clean up
+        for file in [DEFAULT_VERIFY_LOG_NAME, CUSTOM_VERIFY_LOG_NAME]:
+            delete_file(file)
+
+
+def test_truenas_verify_install_log():
+    """
+    The audit subsystem generates an 'as-installed' log
+    This should be clean with no detections or errors
+    """
+    with Client() as c:
+        current_version = c.call('system.version')
+        with open(f"/var/log/audit/truenas_verify.{current_version}.log", 'r') as f:
+            # The last entry is an empty string
+            tnv_init = f.read().splitlines()[:-1]
+
+        # A clean system will have only one entry: The header.
+        assert len(tnv_init) == 1, f"Found unexpected entries\n{tnv_init}"
+
+
+@pytest.mark.parametrize(
+    'type,params', [
+        ("default", []),
+        ("init", ["init", CUSTOM_NAME]),
+        ("syslog", ["syslog"])
+    ],
+    ids=["default", "init", "syslog"]
+)
+def test_truenas_verify(verify_setup, type, params):
+    """
+    Exercize and confirm the three call types: default, init, syslog
```

Comment:
Had to check, these days the American spelling is the same as British Isles: exercise :-)